src/rustzcash.rs

```diff
@@ -693,7 +695,7 @@ pub extern "system" fn librustzcash_sapling_check_output(
     // Accumulate the value commitment in the context
     {
         let mut tmp = cv.clone();
-        tmp.negate(); // Outputs subtract from the total.
+        tmp = tmp.negate(); // Outputs subtract from the total.
```

Comment:
This fixes a bug in the consensus rules, which will result in a fork of testnet by anyone using master before 2.0.0 is released. That's okay, because we'll be rewinding testnet anyway to switch to the final Sapling parameters.